src/main/resources/data/aquaculture/advancements/recipes/misc/gold_nugget_from_gold_fish.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parent": "minecraft:recipes/root",
+  "rewards": {
+    "recipes": [
+      "aquaculture:gold_nugget_from_gold_fish"
+    ]
+  },
+  "criteria": {
+    "has_items": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "item": "aquaculture:goldfish
+          }
+        ]
+      }
+    },
+    "has_the_recipe": {

```

Comment:
Seems like gold_nugget_from_gold_fish have a formatting issue starting here.
Otherwise the rest looks fine :)

Reply:
I don't see any issues there, though I found a typo in the "has_items" criterion

Reply:
Looks like you found it :)